pennylane/operation.py

```diff
@@ -129,6 +129,100 @@
 
 from .utils import pauli_eigs
 
+
+def expand_matrix(base_matrix, wires, wire_order):
+    """Re-express a base matrix acting on a subspace defined by a set of wire labels
+    according to a global wire order.
+
+    .. note::
+
+        This function has essentially the same behaviour as :func:`.utils.expand` but is fully
+        differentiable.
+
```

Comment:
Minor: before merging, could you also add a similar `.. note::` to the `utils.expand` docstring?